src/classes/ACCT_IndividualAccounts_TDTM.cls

```diff
@@ -394,16 +400,18 @@ public class ACCT_IndividualAccounts_TDTM extends TDTM_Runnable {
                 contactsChangedLastName.add(c);
             }
 
-
-            //If Contact Name or Exclude_from_Household_Name__c value has changed
-            if ((conLastNameChanged ||
-                 conFirstNameChanged ||
-                 conExcludeFromHHNameChanged) &&
-                 numOfContactsLeft != 0 &&
-                 conAccountIdNotNull &&
-                 checkHouseholdNaming(c.Account.RecordTypeId))
-            {
-                accountIdToRename.add(c.AccountId);
+            //If Contact Name or Exclude_from_Household_Name__c or Deceased value has changed
+            //and numOfContactsLeft is not 0, rename the Account record
+            if (conAccountIdNotNull && checkHouseholdNaming(c.Account.RecordTypeId)) {
+                if (((conLastNameChanged ||
+                      conFirstNameChanged ||
+                      conExcludeFromHHNameChanged ||
+                      conDeceasedHasChanged) &&
+                 	  numOfContactsLeft != 0)
```

Comment:
Is this indentation on purpose? I fixed one earlier but I worry that the online editor may do bad things :)

Reply:
nope not on purpose